src/views/somns.R

```diff
@@ -515,10 +527,17 @@ perf_diff_table_es <- function(data_es, stats_es, warmup_es, profiles_es, start_
         }
         out('</td>\n')
       }
-      
-      out('<td><button type="button" class="btn btn-sm btn-cmdline" data-content="<code>', cmdline, '</code>"></button>\n')
 
-      warmup_ea <- warmup_es %>%
+      out('<td><button type="button" class="btn btn-sm btn-cmdline" data-content="<code>', cmdline, '</code>"></button>\n')
+      out('<button type="button" class="btn btn-sm btn-environment" data-toggle="popover" data-placement="top" data-content="',environmentStr,'" ></button>')
+      
+      # script to implement the popover for the environment button
+      out('<script>
+          $(document).ready(function(){
+          $(', paste0( "'", '[data-toggle="popover"]' , "'" ) ,').popover();   
+          });
+          </script>')
```

Comment:
Now you have one script per button, but I don't see anything specific for it. Could we do this together with the command line button popover bit?